src/tribol/mesh/MfemData.hpp

```diff
@@ -1572,11 +1621,21 @@ class MfemJacobianData {
    */
   mfem::Array<int> block_offsets_;
 
+  /**
+   * @brief Array of offsets equal to number of displacement degrees of freedom
```

Comment:
Can you clarify? Is this displacement degrees of freedom on the parent mesh? submesh? or redecomp mesh? Wouldn't the displacement dofs just be number of redecomp mesh nodes * dimension of the problem? If there are stacked x, then y, and then z displacement data, the offset is just the number of nodes? I feel like I'm missing something.